src/main/java/org/openhab/binding/zwave/handler/ZWaveControllerHandler.java

```diff
@@ -119,11 +119,11 @@ public void initialize() {
             wakeupDefaultPeriod = 0;
         }
 
-        param = getConfig().get(CONFIGURATION_MAXBATTERYWAKEDURATION);
+        param = getConfig().get(CONFIGURATION_MAXAWAKEPERIOD);
         if (param instanceof BigDecimal) {
-            maxbatteryawake = ((BigDecimal) param).intValue();
+            maxAwakePeriod = ((BigDecimal) param).intValue();
         } else {
-            maxbatteryawake = 5;
+            maxAwakePeriod = 5;
```

Comment:
Can you check to see what happens if you remove this completely? I think with the recent changes to core that configuration should be set to the defaults if a default is set in the configuration definition.  If so, it would be best to only define the default in one place (ie the XML) - if not, we should move this default definition to a constant and define it at the top of the file.